### DISCUSSION TITLE
Why SVG icons not work?

### DISCUSSION THREAD

**Discussion by yoursaf** [fvwmorg/fvwm3#1025] [Q&A]
When I upgrade to fvwm3,all SVG format icons can't display.
`+   "%google-chrome.svg%&Chrome"		    Exec exec $[fvwm_browser]`
or 
`+   "%utilities-terminal.svg:$[icon_size]%&Urxvt"  Exec exec urxvt`
![下载](https://github.com/fvwmorg/fvwm3/assets/6297929/1af1b198-c12a-4f32-838d-cd3b513472bd)
those 2 format code all not work.
Please help me fix this issue.
TYVM

**Comment by ONykyf** [comment]
Because there is a bug that prevents an SVG icon from loading if it lacks specified viewport. A patch that solves it is attached.
[fvwm3-1.1.2-rsvg-dimensions.patch.gz](https://github.com/user-attachments/files/19778772/fvwm3-1.1.2-rsvg-dimensions.patch.gz)
